web/index.html

```diff
@@ -70,287 +68,350 @@ <h3 class="sub-header">
-                                                    {{ data.index[recipe.B][0] }}&nbsp;{{ data.index[recipe.B][1] }}
-                                                </td>
-                                                <td>=</td>
-                                                <td class="item" :class="{'item-used': !saveData.includes(recipe.C)}"
-                                                    @click="navigateTo(recipe.C);">
-                                                    {{ data.index[recipe.C][0] }}&nbsp;{{ data.index[recipe.C][1] }}
-                                                </td>
-                                            </tr>
-                                            <tr v-for="recipe in currentHiddenFrom" v-if="showHiddenRecipes">
-                                                <td class="item" :class="{'item-used': !saveData.includes(recipe.A)}"
-                                                    @click="navigateTo(recipe.A);">
-                                                    {{ data.index[recipe.A][0] }}&nbsp;{{ data.index[recipe.A][1] }}
-                                                </td>
-                                                <td>+</td>
-                                                <td class="item" :class="{'item-used': !saveData.includes(recipe.B)}"
-                                                    @click="navigateTo(recipe.B);">
-                                                    {{ data.index[recipe.B][0] }}&nbsp;{{ data.index[recipe.B][1] }}
-                                                </td>
-                                                <td>=</td>
-                                                <td class="item" :class="{'item-used': !saveData.includes(recipe.C)}"
-                                                    @click="navigateTo(recipe.C);">
-                                                    {{ data.index[recipe.C][0] }}&nbsp;{{ data.index[recipe.C][1] }}
-                                                </td>
-                                            </tr>
-                                        </table>
-                                    </transition>
-                                    <div v-else>
-                                        No recipes found
-                                    </div>
-                                    <div v-if="currentHiddenFromLength > 0">
-                                        <div v-if="!showHiddenRecipes" class="text-link"
-                                            @click="showHiddenRecipes = true">
-                                            {{currentHiddenFromLength}} entries hidden
-                                        </div>
-                                        <div v-else class="text-link" @click="showHiddenRecipes = false">
-                                            {{currentHiddenFromLength}} entries shown
+                                    <transition name="slide-right" mode="out-in">
```

Comment:
Added addition transitions around all of the lists, that allow for a loading data placeholder to be inserted if the content is not ready yet. This allows user to navigate to any pages instantly without waiting, and the data will show up when it is ready.